.github/workflows/module-requests.yml

```diff
@@ -0,0 +1,154 @@
+name: Module Request Management
+
+on:
+  # Individual request analysis
+  issue_comment:
+    types: [created]
+  issues:
+    types: [opened]
+
+  # Weekly bulk triage
+  schedule:
+    - cron: "0 9 * * 1" # Mondays at 9 AM UTC
```

Comment:
Should we comment this out to begin with, until we've tested a bit with `workflow_dispatch`?